orangecontrib/network/widgets/OWNxGenerator.py

```diff
@@ -13,9 +13,20 @@
 
 
 def _balanced_tree(n):
-    b = max(np.log(n) / 2, 2)
-    h = np.log((n * (b - 1)) + 1)/np.log(b) - 1
-    return nx.balanced_tree(int(b), int(h))
+    series = lambda r, h: sum([math.pow(r, x) for x in range(h + 1)])
```

Comment:
Any reference / comments to what all this is?

Reply:
Done!